packages/example-log-extension/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 StrongLoop and IBM API Connect
```

Comment:
2018. I believe all the other packages use a slightly different header for LICENSE. I think this should use the same. 

```
Copyright (c) IBM Corp. 2017,2018. All Rights Reserved.
Node module: @loopback/example-log-extension
This project is licensed under the MIT License, full text below.

--------

MIT license
```

Reply:
Good catch, I'll use `LICENSE` from getting-started example.